qa/pull-tester/rpc-tests.py

```diff
@@ -29,26 +30,22 @@
 import tempfile
 import re
 
-sys.path.append("qa/pull-tester/")
-from tests_config import *
-
 BOLD = ("","")
 if os.name == 'posix':
     # primitive formatting on supported
     # terminal via ANSI escape sequences:
     BOLD = ('\033[0m', '\033[1m')
 
-RPC_TESTS_DIR = SRCDIR + '/qa/rpc-tests/'
+# Read config generated by configure.
+config = configparser.ConfigParser()
+config.read_file(open(os.path.dirname(__file__) + "/tests_config.ini"))
+
+ENABLE_WALLET = config["components"]["ENABLE_WALLET"] == "True"
```

Comment:
yep - I set this as a global variable to match the existing layout. It's been moved to be a local variable in the subsequent refactor commit.

I could remove the variable entirely and access the config directly in the if test, but I don't think it makes much difference at this point.